common/unit_test/Test_Helpers.cpp

```diff
@@ -387,4 +387,4 @@ TEST_P(FFTShiftParamTests, 2DShift2DView) {
 }
 
 INSTANTIATE_TEST_SUITE_P(FFTShift, FFTShiftParamTests,
-                         ::testing::Values(9, 10));
\ No newline at end of file
+                         ::testing::Values(9, 10));
```

Comment:
Could you add https://clang.llvm.org/docs/ClangFormatStyleOptions.html#insertnewlineateof to .clang-format ?